json-server/routes.json

```diff
@@ -7,9 +7,12 @@
     "/users/current": "/users",
     "/deputies/:deputyId/ecm": "/ecm",
     "/deputies/:id/contact-details": "/deputies/:id",
-    "/deputies/:id/assurance-visits": "/assuranceVisit",
+    "/deputies/:id/assurance-visits": "/assuranceVisits",
+    "/deputies/:id/assurance-visits/:visitId": "/assuranceVisit",
     "/firm": "/addFirm",
     "/reference-data/annualBillingInvoice": "/annualBillingInvoice",
     "/reference-data/deputyBooleanType": "/deputyBooleanType",
-    "/reference-data/deputyReportSystem": "/deputyReportSystem"
+    "/reference-data/deputyReportSystem": "/deputyReportSystem",
+    "/reference-data/visitOutcome": "/visitOutcome",
+    "/reference-data/ragRating": "/ragRating"
```

Comment:
This isn't how we should be doing routing. The routes file is just for custom routes. All of this reference data could be in the same object for instance.